### ISSUE TITLE
why 404?

### ISSUE THREAD

**Issue by windkwbs** [ollama/ollama#5294]
![微信截图_20240626152830](https://github.com/ollama/ollama/assets/129468439/23e316b8-cb87-4783-81af-96b94690a61a)

**Comment by d-kleine** [comment]
Have you tried with `http://localhost:11434/v1` like here: https://ollama.com/blog/openai-compatibility

**Comment by windkwbs** [comment]
It still doesn't work. Does this mean Windows systems can only use POST and not support OPTIONS?

**Comment by d-kleine** [comment]
No, should work with Windows too (HTTP methods should be OS-agnostic), must be a different problem. Maybe your port is blocked by a firewall or AV. You could try debugging with Postman (there is a OPTIONS method).

**Comment by d-kleine** [comment]
I just have take a look into my console when doing inference with Ollama, it's only sending POST requests to "api/chat"
